.github/workflows/lint-sample.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Run linter
         uses: golangci/golangci-lint-action@v7
         with:
-          install-mode: goinstall
+          # no install mode needed in v2
```

Comment:
That is great 🎉 
Can you please remove the comment so that we can get it merged?